data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,34 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4105508"
+    name = "pr_cianorte"
+    allowed_domains = ["cianorte.pr.gov.br"]
+    start_urls = ["https://cianorte.pr.gov.br/orgao-oficial/"]
+    current_page = 1
+    base_url = "https://cianorte.pr.gov.br/orgao-oficial/pagina/"
+
+    def parse(self, response):
+        lines = response.css("tr")[1:]
+        if not lines:
+        	return
+        	
+        for row in lines:
+        ...
+                date = row.xpath("td[2]/text()").get()
+                edition_number = row.xpath("td[1]/text()").get()
+
+                yield Gazette(
+                    date=parse(date, languages=["pt"]).date(),
```

Comment:
**suggestion (non-blocking)**: As the format of the dates is stable and consistent (DD/MM/YYYY), you could simply use Python standard library built-in [datetime.strptime](https://docs.python.org/3/library/datetime.html#datetime.datetime.strptime):

`date=datetime.strptime(date, "%d/%m/%Y")`

`dateparser` is an wonderful library, but it is not perfect and is more useful if you don't have a well-defined pattern for the dates you are handling.